packages/server/test/unit/socket_spec.js

```diff
@@ -28,6 +31,19 @@ describe('lib/socket', () => {
       path: 'path-to-browser-one',
     }
 
+    // needed to run these tests locally
+    // sinon.stub(ctx.browser, 'machineBrowsers').resolves([
```

Comment:
I did not test what other impact this has on the test, but frankly I am tired of trying to run these and they fail locally because the electron resolved browser has an empty version string and therefore fails config validation. I would also say this is mostly an integration test, as it integrates heavily with the `data-context` package and the `errors` package